### PR TITLE
GEN-1210 - feat(carDealership): added global banner

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Display banner based on CMS setting;

Screenshot 2023-09-29 at 14 51 17
Screenshot 2023-09-29 at 14 51 32

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3292** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3292" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3292?utm_source=stack-comment).

File: apps/store/src/components/GlobalBanner/GlobalBanner.tsx [hunk 1]
```diff
@@ -25,6 +26,10 @@ const Ellipsis = styled.span({
   overflow: 'hidden',
   textOverflow: 'ellipsis',
   whiteSpace: 'nowrap',
+
+  '& > b': {
```
Comment (robinandeer) [inline apps/store/src/components/GlobalBanner/GlobalBanner.tsx:30]: aha smart ;)

File: apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx [hunk 2]
```diff
@@ -12,7 +16,32 @@ type Props = SbBaseBlockProps<{
...
+  const { t } = useTranslation('cart')
+
+  const data = useCarTrialQuery({
+    onCompleted(data) {
+      const requirePaymentConnection = props.blok.requirePaymentConnection ?? false
+
+      if (requirePaymentConnection) {
+        const today = new Date()
+        const dueDate = new Date(data.trialContract.startDate)
+        dueDate.setDate(dueDate.getDate() + 14)
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx:30]: nit: move this number into a named variable

File: apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx [hunk 3]
```diff
@@ -34,25 +63,33 @@ export const CarTrialExtensionBlock = (props: Props) => {
 }
 CarTrialExtensionBlock.blockName = 'carTrialExtension'
 
-const useCarTrialQuery = () => {
+type UseCarTrialQueryOptions = Pick<QueryHookOptions<TrialExtension>, 'onCompleted'>
+
+const useCarTrialQuery = (options: UseCarTrialQueryOptions) => {
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx:68]: nit: we usually name this `params`

File: apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx [hunk 4]
```diff
@@ -34,25 +63,33 @@ export const CarTrialExtensionBlock = (props: Props) => {
...
   const router = useRouter()
   const queryParam = router.query['id']
   const id = typeof queryParam === 'string' ? queryParam : undefined
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx:71]: nit: name this "shopSessionId"

**Review by robinandeer — APPROVED** [review]
let's try it out!

**Pull request merged**